frontend/src/components/pages/RequestKit.jsx

```diff
@@ -69,11 +67,16 @@ function RequestKit({ user }) {
         alert("Your order has been processed.");
 
         // Clear input fields
-        setPhone("");
-        setSchoolName("");
-        setSchoolYear("");
-        setSchoolAddress("");
-        setComments("");
+        setOrderForm({
+          phone: "",
+          schoolName: "",
+          schoolAddress: "",
+          comments: "",
+          schoolYear: "",
+          email: user ? user.email : "",
+          firstName: user ? user.first_name : "",
+          lastName: user ? user.last_name : "",
```

Comment:
Hmm. I actually don't think this action is necessary. You're navigating them away: why bother clearing the form?

I'd remove this.